constants.go

```diff
@@ -23,6 +23,7 @@ const (
 	MIMETextCSS         = "text/css"
 	MIMEApplicationXML  = "application/xml"
 	MIMEApplicationJSON = "application/json"
+	MIMEApplicationCBOR = "application/cbor"
```

Comment:
@imsk17 pls add it to https://github.com/gofiber/fiber/blob/main/docs/api/constants.md#mime-types-that-are-commonly-used

Reply:
solved with [`fc1779b` (#3173)](https://github.com/gofiber/fiber/pull/3173/commits/fc1779b6df4602e0acd42e5695fe9cda7d439880)